gradle.properties

```diff
@@ -1,8 +1,8 @@
-korkVersion=7.106.0
+korkVersion=7.150.0
+kotlinVersion=1.6.21
 org.gradle.parallel=true
-spinnakerGradleVersion=8.10.1
+spinnakerGradleVersion=8.23.0
```

Comment:
Would you mind trying the recently released 8.25.0?  I suppose we can bump it in a subsequent PR too...

Reply:
Actually, never mind.  There's https://github.com/spinnaker/swabbie/pull/580 which we can rebase once this goes in.